docs/manual/layout/templates/twig/namespace.en.md

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Namespaces'
```

Comment:
I don't think we should cover namespaces in the user manual. At least not in detail. What people *should* know probably boils down to the following:

* The full template name inside Twig contains a "namespace". And a namespace looks like `@Something`.
* Our Contao templates are all inside the `@Contao` namespace, so that's why for instance the text content element's "fully qualified template name" is `@Contao/content_element/text.html.twig` and why you would extend from it with `{% extends "@Contao/content_element/text.html.twig" %}`.
* The `@Contao` namespace is a "managed namespace" and has a special feature in contrast to standard Twig: The same template can be extended from various sources. This way an extension can for instance add a new feature to a core template while you can still adjust it in your application. And maybe a word about who comes first.

Reply:
Done